api/chains/api.go

```diff
@@ -36,6 +36,7 @@ func Register(router *gin.Engine) {
 		GET("/mint/annual_provisions", GetAnnualProvisions).
 		GET("/mint/epoch_provisions", GetEpochProvisions).
 		GET("/staking/params", GetStakingParams).
+		GET("/APR", GetStakingAPR).
```

Comment:
```suggestion
		GET("/apr", GetStakingAPR).
```

I would keep all paths/urls lowercase as a convention